### ISSUE TITLE
Initialise new distributions to a bus factor of 1 and zero dependencies

### ISSUE THREAD

**Issue by neilb** [metacpan/metacpan-api#1025]
When you see a new distribution in the CPAN Index, you could initialise the bus factor to 1 (by definition, since the person just uploaded it) and the river data to 0 direct dependencies, 0 total dependencies, bucket 0.

That way when people look at the recent page, for example, new distributions won't stand out, and everything will have river and bus factor data available.

**Comment by mickeyn** [comment]
@neilb: the default values are now in place (or at least the code is merged).
We currently have 1343 distributions without the river data. Would you like use to back-fill them (with the defaults?)?

**Comment by neilb** [comment]
Thanks @mickeyn. Could you send me a list of those distributions please, so I can check whether my code is missing them, or something else is going on?

There are some that I know my script skips because they've got broken metadata files, or no metadata, but that's a lot more than I'd have expected.

**Comment by mickeyn** [comment]
Run this:
```
curl 'https://fastapi.metacpan.org/distribution/_search?size=1500&fields=' -d'{"query":{"bool":{"must_not":{"exists":{"field":"river"}}}}}' | grep '"_id" :' | awk -F: '{gsub(/[ ,"]/,""); print $2}' | sort
```
:)

**Comment by oalders** [comment]
Closed via #1027

**Comment by mickeyn** [comment]
@neilb if you still need data corrected - please open a separate issue and I'll fix it.
